### ISSUE TITLE
Add option to use "offline UUID" for new players instead of random UUIDv4

### ISSUE THREAD

**Issue by evan-goode** [unmojang/drasl#99]
There are potentially many users converting an existing offline server/group of servers to use Drasl. Rather than generating a random UUID for new accounts, it may make more sense to just use the player's offline UUID.

There should be a configurable option for this behavior and it should maybe be the default.

**Comment by anatolykopyl** [comment]
Another potentially useful feature would be for the default player autogenerated in each user to have the same UUID as the user.
I am creating users for my servers via API and if I want to set the player UUIDs I have to delete the player and re-create it for each user, since there is no way to set the player UUID with the create user request.

**Comment by evan-goode** [comment]
Yeah, perhaps. The option should certainly be an enum (represented as a string in TOML) rather than a handful of booleans. `PlayerUUIDGeneration` or something.

> there is no way to set the player UUID with the create user request.

Well, there is `chosenUuid` on `APICreateUserRequest`, but in your case of wanting to match the user UUID, you couldn't pass the UUID of the user since you don't know it yet.

**Comment by Blasg2** [comment]
sorry to ask here but idk the right place: Is there a way to keep my server offline, but the offline accounts arent able to use Nicks registered on drasl. Or maybe just make users registered on drasl not have to use the Auth mod password that offline users will have to use everytime they log
